pkgs/development/python-modules/aider-chat/default.nix

```
@@ -0,0 +1,109 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, pythonOlder
+, universal-ctags
+, aiohttp
+, aiosignal
+, async-timeout
+, attrs
+, backoff
+, certifi
+, charset-normalizer
+, configargparse
+, diskcache
+, frozenlist
+, gitdb
+, gitpython
+, idna
+, jsonschema
+, markdown-it-py
+, mdurl
+, multidict
+, networkx
+, numpy
+, openai
+, packaging
+, prompt-toolkit
+, pygments
+, pyyaml
+, requests
+, rich
+, scipy
+, smmap
+, sounddevice
+, soundfile
+, tiktoken
+, tqdm
+, urllib3
+, wcwidth
+, yarl
+}:
+
+buildPythonPackage rec {
+  pname = "aider-chat";
+  version = "0.14.1";
+
+  disabled = pythonOlder "3.8";
```

See https://github.com/paul-gauthier/aider/blob/v0.14.1/setup.py#L21

```suggestion
  disabled = pythonOlder "3.9";
```